### ISSUE TITLE
Notification service settings not respected

### ISSUE THREAD

**Issue by DidierLmn** [jellyfin/jellyfin#3092]
**Describe the bug**
Notification service settings net respected. When service is disabled (not checked) for a notification type, the service is still used to send the notification. (In case of email at least)

**System (please complete the following information):**
 - OS: Fedora 32
 - Jellyfin Version: 10.5.5
 - Installed Plugins: Gotify Notifications, Email Notifications
 - Reverse Proxy: nginx
 - Base URL: none

**To Reproduce**

1. Install Gotify notifications
2. Configure Gotify notifications
3. Enable Gotify notifications for user
4. Install Email notifications
5. Configure Email notifications
6. Enable Email notifications for user
7. Enable "Video Playback started" notification. 
8. Check "Gotify" and uncheck "Email" in "Video Playback started" notification settings
9. Start video Playback
10. Both email and gotify notification is being sent

**Expected behavior**
No email notification is being sent.

**Logs**
No errors in logs, only INFO messages
```
[2020-05-11 07:30:44.288 +02:00] [INF] [40] PlaybackReporting - EventMonitorEntryPoint: Adding playback tracker : b6ee4e9133134194-f9995879e04541d6b154580e3c86f4e7-d81ad47c5738e180b552e22084a9d374
[2020-05-11 07:30:44.289 +02:00] [INF] [40] PlaybackReporting - EventMonitorEntryPoint: PlaybackTracker : Adding Start Event : 05/11/2020 07:30:44
[2020-05-11 07:30:44.289 +02:00] [INF] [40] PlaybackReporting - EventMonitorEntryPoint: Creating StartPlaybackTimer Task
[2020-05-11 07:30:44.292 +02:00] [INF] [46] PlaybackReporting - EventMonitorEntryPoint: StartPlaybackTimer : Entered
[2020-05-11 07:30:44.353 +02:00] [INF] [40] App: Sending email "xxxxx" with subject "Emby: xxx has started playing xxxxx"
[2020-05-11 07:30:45.081 +02:00] [INF] [28] App: Completed sending email "xxx" with subject "Emby: xxx has started playing xxx"
[2020-05-11 07:30:45.210 +02:00] [INF] [40] PlaybackReporting - EventMonitorEntryPoint: Processing playback tracker : "b6ee4e9133134194-f9995879e04541d6b154580e3c86f4e7-d81ad47c5738e180b552e22084a9d374"
```

**Additional context**
Even when disabling all notification services for a notification, the notification is still being sent to all notifications services.

**Screenshots**
<img width="874" alt="Screenshot 2020-05-11 at 07 48 37" src="https://user-images.githubusercontent.com/15478968/81528036-1497fa00-935c-11ea-8a71-3a622e33ff93.png">

**Comment by eltiel** [comment]
Likely related to https://github.com/jellyfin/jellyfin/issues/2262

**Comment by VigibotDev** [comment]
Why it's closed ? 1.8.0 show me the bug just now. Debian 11 Server, Debian 11 Client...
